### ISSUE TITLE
how to run html5 boilerplate?

### ISSUE THREAD

**Issue by miro0307** [h5bp/html5-boilerplate#2049]
hi，this is my first time using html5-boilerplate,
I download in homepage，version 6.1.0

and then I run command npm install
but i can not find a scripts like 'npm run start' and can not know how to run this project
did i do something wrong?

**Comment by snolangps** [comment]
yeah, but how do you `run` the application to see it up and running? There's no documentation on this...

**Comment by roblarsen** [comment]
@snolangps There's no documentation on how to run it, because there's nothing to run. It's just a collection of HTML, CSS and JavaScript. If you click on [index.html](https://github.com/h5bp/html5-boilerplate/blob/master/dist/index.html) in the [/dist/](https://github.com/h5bp/html5-boilerplate/blob/master/dist/) folder and open it in a web browser you're running HTML5 Boilerplate.
